contracts/acl/ACL.sol

```diff
@@ -156,6 +157,18 @@ contract ACL is IACL, TimeHelpers, AragonApp, ACLHelpers {
         _setPermissionManager(address(0), _app, _role);
     }
 
+    /**
+    * @notice Lock `_role` in `_app`, so no midification can be made to it (grant, revoke, permission manager)
+    * @param _app Address of the app in which the permission is being unmanaged
+    * @param _role Identifier for the group of actions being unmanaged
+    */
+    function petrifyPermission(address _app, bytes32 _role)
+        external
+        onlyPermissionManager(_app, _role)
```

Comment:
Thinking about whether this action should be done by the manager, or also allow to do it on permissions that haven't been created yet. It feels a bit off that you would need to create a permission and then immediately lock it, as after creating the permission you would have to revoke it from the entity that got the permission so it is completely clean and no one has the permission.

Reply:
You can create the permission using `LOCK_ENTITY` (i.e., `address(1)`) as manager and granted but it's certainly not so clean, so yes, changing it.

Reply:
Btw, would it make sense to add helpers/wrappers like `createLockedPermission` or `createEmptyLockedPermission`? (they would pass `LOCK_ENTITY` as granted or manager)